server/models/dummyData/generateListingsArray.js

```diff
@@ -0,0 +1,27 @@
+const faker = require('faker');
+
+////////// POSTGRES ONLY //////////
+
+const generateDummyArray = () => {
+  return new Promise((resolve, reject) => {
+    var scaleListingsArray = [];
+    for (var j = 0; j < 10000; j++) {
```

Comment:
10000 could assigned to and used as const NUM_ARRAYS (or something) for readability

Reply:
good note, thanks!